### PR TITLE
Fix/docker push latest

### PR DESCRIPTION
**Pull request by andrewcharnley**
Allows tagging of latest
Allows seed data to continue on conflicts
Allows localdev to point to correct docker compose network

### CODE REVIEW

**Review by Obsiye — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -218,7 +218,8 @@ jobs:
                 command: |
                   docker login -u="${QUAYIO_USERNAME}" -p="${QUAYIO_PASSWORD}" quay.io
                   docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
-                  docker push quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
+                  docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:latest
+                  docker push --all-tags quay.io/hmpps/<<parameters.app_name>>
```
Comment (Obsiye) [inline .circleci/config.yml:222]: maybe you have to do `docker image push --all-tags quay.io/hmpps/<<parameters.app_name>>` as i can see the tests erroring on the flag --all-tags

**Review by Obsiye — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (andrewcharnley) [reply .circleci/config.yml:222]: It's building using circle:python:3 which I assume takes the latest 3.x.x, and 20.10.0 came in -

https://circleci.com/developer/images/image/cimg/python:3.8.9

So I'd expect it to be in...

Comment (andrewcharnley) [reply .circleci/config.yml:222]: Server Engine Details:
  Version:          24.0.9

Odd!

Comment (Obsiye) [reply .circleci/config.yml:222]: Maybe try `docker image push ...`

**Pull request closed without merging**